docs-ref-conceptual/microsoft-graph-migration.md

```diff
@@ -36,6 +38,7 @@ Command argument and behavior breaking changes are listed below.
 ### `az ad app permission grant`
 
 - Remove `--expires`
+- `--scope` doesn't default to `user_impersonation` anymore and is now required.
```

Comment:
double negative.  :-)  change to 

no longer defaults to `user_impersonation` and is now required

Reply:
Thanks @dbradish-microsoft, this is definitely more concise, though I don't think anymore is a "negative". 🤣 https://ludwig.guru/s/don't+do+that+anymore

```suggestion
- `--scope` no longer defaults to `user_impersonation` and is now required
```